src/AutopinPlus/Monitor/PageMigrate/Main.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is part of Autopin+.
+ * Copyright (C) 2015 Technische Universität München - LRR
+ *
+ * This file is licensed under the GNU General Public License Version 3
+ */
+
+#include <AutopinPlus/Monitor/PageMigrate/Main.h>
+
+#include <QTime>
+#include <stdlib.h>
+extern "C" {
```

Comment:
Could you add

#ifdef __cplusplus
extern "C" {
#endif
....
#ifdef __cplusplus
}
#endif

to the header file and remove the extern "C" here?